paymentsheet-example/src/androidTest/java/com/stripe/android/lpm/TestSepaDebit.kt

```diff
@@ -130,6 +131,10 @@ internal class TestSepaDebit : BasePlaygroundTest() {
         )
     }
 
+    private fun PlaygroundState?.asPaymentState(): PlaygroundState.Payment? {
```

Comment:
This is used in at least a few places, should we promote it out of private in these files?